ckanext/versions/logic/action.py

```diff
@@ -18,14 +18,14 @@
 def version_update(context, data_dict):
     """Update a version from the current dataset.
 
-    :param dataset: the id or name of the dataset
-    :type dataset: string
-    :param version: the id of the version
-    :type version: string
+    :param package_id: the id the dataset
+    :type package_id: string
+    :param version_id: the id of the version
+    :type version_id: string
     :param name: A short name for the version
     :type name: string
-    :param description: A description for the version
-    :type description: string
+    :param notes: A description for the version
```

Comment:
If it's not too late I would keep `description`. `notes` is just a legacy name on the dataset field (and a confusing one for people).

If it's too much hassle don't worry.